### PR TITLE
Sleep faster

### PR DESCRIPTION
**Pull request by Horneth**
- Enables parallel execution of all tests
- Sleeps 2 seconds after every task writing something to give some time to the VM to flush its file system before completing the task

### CODE REVIEW

File: test_cromwell.sh [hunk 1]
```diff
@@ -112,14 +116,18 @@ fi
 cd centaur
 TEST_STATUS="failed"
 
+sbt test:compile
+CP=$(sbt "export test:dependency-classpath" --error)
+
 if [[ -n ${EXCLUDE_TAG} ]]; then
     echo "Running Centaur filtering out ${EXCLUDE_TAG} tests"
-    TEST_COMMAND="sbt ${REFRESH_TOKEN} \"test-only -- -l ${EXCLUDE_TAG}\""
+    TEST_COMMAND="java ${REFRESH_TOKEN} -cp $CP org.scalatest.tools.Runner -R target/scala-2.11/test-classes -oD -PS${TEST_THREAD_COUNT} -l ${EXCLUDE_TAG}"
```
Comment (geoffjentry) [inline test_cromwell.sh:124]: I thought you could do this directly with SBT somehow but I can't find it now. Not a huge deal but it'd be nice to maintain the sbt invocation, but not worth tracking down I guess

**Comment by geoffjentry** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/118/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

File: test_cromwell.sh (hunk 1 above)
Comment (Horneth) [reply test_cromwell.sh:124]: Yeah I pretty much copy pasted this from the `run_tests_parallel.sh` script in centaur.

**Comment by ruchim** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/118/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Pull request merged**